src/praisonai/tests/unit/integrations/test_managed_agents.py

```
@@ -124,74 +131,95 @@ async def stream(self, prompt: str, **kwargs) -> AsyncIterator[Dict[str, Any]]:
         
-        # Create without explicit API key to trigger env lookup
-        managed = ManagedAgentIntegration(provider="anthropic", api_key=None)
+        # Verify we got a meaningful response
+        assert isinstance(result, str)
+        assert len(result.strip()) > 0
+        assert "hello" in result.lower() or "hi" in result.lower()
         
-        # Should have stored the env key back to api_key
-        assert managed.api_key == "env_api_key"
\ No newline at end of file
+    except Exception as e:
```

Ruff (0.15.10)

[warning] Do not catch blind exception: `Exception`

(BLE001)